gatsby-browser.js

```diff
@@ -11,7 +11,7 @@ export const wrapRootElement = ({ element }) => (
 export const onClientEntry = () => {
   window[`ga-disable-${process.env.GOOGLE_ANALYTICS_ID}`] = true
 
-  if (Cookie.get('accept_cookies') === 'true') {
+  if (Cookie.get('accept_cookies') === 'true' && Cookie.get('deny_analytics') === 'false') {
```

Comment:
Since the first cookie is called _accept_cookies_, it would be more intuitive to have the second one named _accept_analytics_